tests/test_rpmutils.py

```diff
@@ -59,8 +59,9 @@ def test_no_component(self):
                 ('', '1.2', '', '', ''))
 
     def test_absurd(self):
-        assert (split_filename('1:aarch64-ppc64le-i686-0.1-1.el7ost.src.rpm') ==
-                ('aarch64-ppc64le-i686', '0.1', '1.el7ost', '1', ''))
+        whole_filename = '1:aarch64-ppc64le-i686-0.1-1.el7ost.src.rpm'
+        split_parts = ('aarch64-ppc64le-i686', '0.1', '1.el7ost', '1', '')
+        assert (split_filename(whole_filename) == split_parts)
```

Comment:
This looks much better, thanks! I notice, however, that the below test case (which I know you didn't change, it was like this) has the same problem.  Ideally, this patch would remove any NOQA markers that can be easily cleaned up, so we can be more confident the code is actually flake8 compliant.  This might require a little bit of rename so the variables are clear, but if you could apply the same treatment to the below chunk as the one above, that would be much appreciated.